Parser/asdl_c.py

```diff
@@ -428,6 +428,17 @@ def simpleSum(self, sum, name):
     def buildArgs(self, fields):
         return ", ".join(fields + ["arena"])
 
+    def typeCheck(self, name):
+        self.emit("tp = astmodulestate_global->%s_type;" % name, 1)
+        self.emit("isinstance = PyObject_IsInstance(obj, tp);", 1)
+        self.emit("if (isinstance == -1) {", 1)
+        self.emit("return 1;", 2)
+        self.emit("} else if (isinstance == 0 && field != NULL) {", 1)
+        error = 'field \\"%%s\\" was expecting %s type node, got %%s' % name
```

Comment:
```suggestion
        error = "field '%%s' was expecting '%s' type node, got '%%s'" % name
```

Reply:
that was my initial thought too but it will look inconsistent with the other error messages raised by this module (not the `'expr'` part, but the `field 'blabla'` part)
```
TypeError: required field "lineno" missing from expr
```

Reply:
CPython, in general, uses single quotes in error messages, so I would prefer to stick to this convention in all places.